Taskfile.yaml

```diff
@@ -179,3 +180,7 @@ tasks:
   pre-commit-run:
     cmds:
       - pre-commit run --all-files || pre-commit run --all-files
+  docs:
+    cmds:
+      - npx serve -l 4001 &
+      - cd frontend/docs/ && pnpm run dev
```

Comment:
important: this should make it a lot easier to run the docs and do the github emoji thingy going forward